ghcide/src/Development/IDE/GHC/Dump.hs

```diff
@@ -21,18 +21,18 @@ import           GhcPlugins
 import           Prelude                         hiding ((<>))
 
 -- | Show a GHC syntax tree in HTML.
-#if MIN_VERSION_ghc(9,2,1)
-showAstDataHtml :: (Data a, ExactPrint a, Outputable a) => a -> SDoc
-#else
 showAstDataHtml :: (Data a, Outputable a) => a -> SDoc
-#endif
 showAstDataHtml a0 = html $
     header $$
     body (tag' [("id",text (show @String "myUL"))] "ul" $ vcat
         [
+-- #if MIN_VERSION_ghc(9,2,1)
+
+-- #else
 #if MIN_VERSION_ghc(9,2,1)
-            li (pre $ text (exactPrint a0)),
-            li (showAstDataHtml' a0),
+--             li (pre $ text (exactPrint a0)),
+--             li (showAstDataHtml' a0),
+--             li (nested "Raw" $ pre $ showAstData NoBlankSrcSpan NoBlankEpAnnotations a0)
```

Comment:
Why are these still comments? Should probably be removed, if we are not going to use them anymore?

Reply:
I want to get @pepeiborra's opinion on this first.